native/cocos/renderer/core/PassUtils.cpp

```diff
@@ -225,6 +194,56 @@ const ccstd::unordered_map<gfx::Type, GFXTypeWriterCallback> type2writer = {
      }},
 };
 
+
+const ccstd::unordered_map<gfx::Type, GFXTypeValidatorCallback> type2validator = {
+    {gfx::Type::UNKNOWN, [](const MaterialProperty & /*v*/) -> bool {
+         CC_LOG_ERROR("type2validator unknown type");
+         return false;
+     }},
+    {gfx::Type::INT, [](const MaterialProperty &v) -> bool {
+         auto *p = ccstd::get_if<int32_t>(&v);
+         return p != nullptr ? true : false;
+     }},
+    {gfx::Type::INT2, [](const MaterialProperty &v) -> bool {
+         auto *p = ccstd::get_if<Vec2>(&v);
+         return p != nullptr ? true : false;
```

Comment:
Please fix the errors clang-tidy reported.

Reply:
Fixed